<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>495</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackedWidget
{
	background-image: url(Pic/back.jpg);
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QQPushButton" name="imageButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>320</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Image detection</string>
     </property>
    </widget>
    <widget class="QQPushButton" name="realButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>230</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Real-time detection</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>851</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#ffffff;
background-color:#adc1d7;</string>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Detection System of Passion Fruit </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageCap">
    <widget class="QQPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QQPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>90</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QQPushButton" name="returnButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>150</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QLabel" name="capLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 1px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(29, 38, 113, 0.5), stop:1e-05 #1d2671, stop: 1.0 rgba(84, 236, 215, 0.5));      
             
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="capBox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>210</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 18px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pagePic">
    <widget class="QLabel" name="picLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 1px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(29, 38, 113, 0.5), stop:1e-05 #1d2671, stop: 1.0 rgba(84, 236, 215, 0.5));      
             
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QQPushButton" name="selectButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>160</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Select Image</string>
     </property>
    </widget>
    <widget class="QQPushButton" name="detectPicButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>220</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Detection</string>
     </property>
    </widget>
    <widget class="QQPushButton" name="takeButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Take Photo</string>
     </property>
    </widget>
    <widget class="QQPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>330</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QQPushButton" name="cutButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>100</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 25px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>Shoting</string>
     </property>
    </widget>
    <widget class="QComboBox" name="picBox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>280</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
                border-radius:5px;
                border-width: 2px;
                color: #d3dae3;
                padding: 5px;
            background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #c33764, stop:1e-05 #1d2671, stop: 1.0 #54ecd7);      
            border-radius: 18px;   
            border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
            border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
            border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
            border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>420</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Sans</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-radius:5px;
border-width: 0px;
color: #ff0000;/*d3dae3*/
padding: 5px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(29, 38, 113, 0.1), stop:1e-05 #1d2671, stop: 1.0 rgba(84, 236, 215, 0.1));         
border-top-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.466, stop: 1e-05  #54ecd7, stop: 1  #1d2671);
border-bottom-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop: 1e-05 #54ecd7, stop: 1  #1d2671);
border-right-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:1e-05  #54ecd7, stop: 1  #1d2671);
border-left-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop: 1e-05 #54ecd7, stop: 1  #1d2671);</string>
     </property>
     <property name="text">
      <string>RunTime:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">custom_buttons</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
